autoware_launch/config/planning/preset/default_preset.yaml

```diff
@@ -0,0 +1,87 @@
+launch:
+  # - arg:
+  #     name: behavior_path_planner_launch_modules
+  #     default: "[behavior_path_planner::Avoidance,
+  #       behavior_path_planner::AvoidanceByLaneChange,
+  #       behavior_path_planner::DynamicAvoidance,
+  #       behavior_path_planner::LaneChange,
+  #       behavior_path_planner::ExternalLaneChange,
+  #       behavior_path_planner::StartPlanner,
+  #       behavior_path_planner::GoalPlanner,
+  #       behavior_path_planner::SideShift]"
+  #     # option: behavior_path_planner::Avoidance
+  #     #         behavior_path_planner::AvoidanceByLaneChange
+  #     #         behavior_path_planner::DynamicAvoidance
+  #     #         behavior_path_planner::LaneChange
+  #     #         behavior_path_planner::ExternalLaneChange
+  #     #         behavior_path_planner::StartPlanner
+  #     #         behavior_path_planner::GoalPlanner
+  #     #         behavior_path_planner::SideShift"
```

Comment:
seems better to comment the reason why comment out. currently not supported?

Reply:
update in https://github.com/autowarefoundation/autoware_launch/pull/680/commits/096acd81328763243f5acbb19636af4d09b673d2.